test/RouteMiddlewareTest.php

```diff
@@ -265,9 +265,9 @@ public function testRoutingSuccessResultingInContainerExceptionReRaisesException
     public function routerAdapters()
     {
         return [
-          'aura'       => [ 'Zend\Expressive\Router\Aura' ],
-          'fast-route' => [ 'Zend\Expressive\Router\FastRoute' ],
-          'zf2'        => [ 'Zend\Expressive\Router\Zf2' ],
+          'aura'       => [ 'Zend\Expressive\Router\AuraRouter' ],
```

Comment:
Héhé, this is why ::class is so much better even in tests. Make refactoring much easier! ;)